src/SvgParser.elm

```diff
@@ -256,67 +262,72 @@ xmlDeclarationParser =
         )
 
 
-{-| Parses `String` to `SvgNode`. Normally you will use `parse` instead of this.
-
-    parse "<svg xmlns=\"http://www.w3.org/2000/svg\"></svg>"
-        == Ok (SvgElement (Element "svg" [ ( "xmlns", "http://www.w3.org/2000/svg" ) ] []))
-
+{-| Same as parseToNode, but returns a list of all the nodes in the string.
 -}
-parseToNode : String -> Result String SvgNode
-parseToNode input =
+parseToNodes : String -> Result String (List SvgNode)
+parseToNodes input =
     case
         Combine.runParser
             (andMapRight
                 (optional "" xmlDeclarationParser)
-                nodeParser
+                (many nodeParser)
             )
             []
             input
     of
-        Ok ( _, _, svgNode ) ->
-            Ok svgNode
+        Ok ( _, _, svgNodes ) ->
+            Ok svgNodes
 
-        Err ( _, stream, errors ) ->
+        Err ( _, _, errors ) ->
             Err <| String.join " or " errors
 
 
-{-| Same as parseToNode, but returns a list of all the nodes in the string.
+{-| Parses `String` to `SvgNode`. Normally you will use `parse` instead of this.
+
+    parse "<svg xmlns=\"http://www.w3.org/2000/svg\"></svg>"
+        == Ok (SvgElement (Element "svg" [ ( "xmlns", "http://www.w3.org/2000/svg" ) ] []))
+
 -}
-parseToNodes : String -> Result String (List SvgNode)
-parseToNodes input =
+parseToNode : String -> Result String SvgNode
+parseToNode input =
     case
         Combine.runParser
             (andMapRight
                 (optional "" xmlDeclarationParser)
-                (many nodeParser)
+                nodeParser
             )
             []
             input
     of
-        Ok ( _, _, svgNodes ) ->
-            Ok svgNodes
+        Ok ( _, _, svgNode ) ->
+            Ok svgNode
 
-        Err ( _, stream, errors ) ->
+        Err ( _, _, errors ) ->
             Err <| String.join " or " errors
 
 
-{-| Parses `String` to `Html msg`. Usually this is the only function you need.
+{-| Parses `String` to `Html msg`. This function filters top level comments to find the first `<svg>` element.
```

Comment:
Can you add something like ``"Additional `<svg>` elements are ignored."``? In the end I am okay with updating the documentation to the new behavior.